archetype-module-platform/pom.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openmrs.maven</groupId>
+    <artifactId>openmrs-sdk</artifactId>
+    <version>3.0.0-SNAPSHOT</version>
+  </parent>
+  
+  <groupId>org.openmrs.maven.archetypes</groupId>

```

Comment:
I am not sure about groupId, leave them as they are or change to the same as parent (`org.openmrs.maven`)?